src/status_im2/subs/wallet/networks.cljs

```diff
@@ -0,0 +1,42 @@
+(ns status-im2.subs.wallet.networks
+  (:require [quo.foundations.resources :as resources]
+            [re-frame.core :as re-frame]
+            [status-im2.constants :as constants]))
+
+(re-frame/reg-sub
+ :wallet/filtered-networks-by-mode
+ :<- [:wallet/networks]
+ (fn [networks [_ test?]]
+   (get networks (if test? :test :prod))))
```

Comment:
From real experience, I'm always a bit wary of the dangers of mixing test and prod networks. I'm wondering if we shouldn't have a global state to define the network mode, and all the code uses that value, so there's less of a chance of some code passing the wrong flag as argument because there'd be no dynamic arg `test?` to the sub, just the value from the app db that all consumers agree on.

I'm not suggesting any change in the PR, but I would personally avoid this if possible.

Reply:
oh the intention is that this will become a sub itself and that will be a global feature in the project. We will have a switch to toggle that on/off. I have some future work to change this from a param to a sub-value from the db to get that data 👍

Reply:
Sounds great @J-Son89. Thanks!